scripts/tools-setup.js

```diff
@@ -0,0 +1,18 @@
+const liquibase = require("liquibase");
+const path = require("path");
+
+// see README > "Integration tests" for why this awful code exists
+(async () => await liquibase({
+    liquibase: path.join(__dirname, "../node_modules/liquibase/lib/liquibase-4.0.0/liquibase"),
+    changeLogFile: path.join(__dirname, "../cvs-nop/changelog-master.xml"),
+    url: 'jdbc:mysql://localhost:3306/CVSBNOP',
+    username: "root",
```

Comment:
Couldn't we add that to environmental variables? I would suspect this would need to be changed in the pipeline based on our environments?
I think for now we seem to be having `local`, `dev`, `integration`, `preprod`, `prod`

We could use an `.env` file, I think the pipeline might have that already (@githanium )?

Reply:
This was added under explicit instructions from @githanium, but can be removed if he says it's OK :)

Reply:
Resolved: file deleted with Tanio's blessing https://github.com/dvsa/cvs-tsk-update-store/pull/1/commits/e8fe927eb34091e84369f214815a38672a2b7f48